src/fragments/lib/restapi/native_common/getting-started/common.mdx

```diff
@@ -61,7 +61,7 @@ import flutter11 from "/src/fragments/lib/restapi/flutter/getting-started/30_ini
 
 ## Make a POST Request
 
-Copy and paste the following code in your application so that it runs only once when the app starts:
+Send a POST request with a JSON body.
```

Comment:
callout: This part applies to android and iOS as well but I thought it made more sense. LMK if anyone disagrees and I can change back.